csharp/src/Ice/Reference.cs

```diff
@@ -1832,17 +1578,19 @@ internal void Write(OutputStream ostr)
             IReadOnlyList<InvocationInterceptor> invocationInterceptors, // already a copy provided by Ice
             InvocationMode invocationMode,
             TimeSpan? invocationTimeout,
+            object? label,
             IReadOnlyList<string> location, // already a copy provided by Ice
             TimeSpan? locatorCacheTimeout,
             LocatorInfo? locatorInfo,
-            bool? preferNonSecure,
+            bool preferExistingConnection,
```

Comment:
should be bool? preferExistingConnection

Reply:
Sorry, I was confused. There is no `Ice.Default.PreferExistingConnection` some a plain bool is fine.